<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hashtagManager</class>
 <widget class="QDialog" name="hashtagManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <widget class="QToolButton" name="fetchCollectionHashtags">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="download">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="requestCollectionMissingTags">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QTableWidget" name="collectionHashtags">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hastag</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Likes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comments</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Engagement</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Favorite</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="collectionEdit"/>
   </item>
   <item row="0" column="0" rowspan="4" colspan="2">
    <widget class="QListWidget" name="collections">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2" rowspan="3" colspan="4">
    <widget class="QTableWidget" name="hashtagTableWidget">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hastag</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Likes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comments</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Engagement</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="hashtagLookup">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="fetchSuggestions">
     <property name="text">
      <string>!!!</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="suggestedSearch"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="removeCollectionButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="addCollectionButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="forceTagsInCollection">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="tag-new"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>hashtagManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>hashtagManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
